libs/sketch/src/lib/components/dialog/dialog.component.css

```diff
@@ -15,15 +15,22 @@
   --sk-dialog-backdrop-blur: 0;
 }
 
+.display-flex {
+  display: flex;
+}
+
 .sk-dialog {
-  width: var(--sk-dialog-width);
-  height: var(--sk-dialog-height);
+  width: 100vw;
```

Comment:
why did we need to replace the variables? Couldn't we use default values for the var() instead?

Reply:
because we moved some things around, found a small styling bug with fullscreen dialog, will fix it.

Reply:
the .sk-dialog is the "container" in the background with the backDrop, so it is always on full screen size,
the actual content is in a nested child and .sk-dialog-fullscreen-dialog will applyed on Fullscreen, rest is adepted to the content itself

Reply:
ah ok, sounds mis-named then?

Reply:
in fullscreen the .sk-dialog has the width of the content which is easily fixed with flex-grow: 1; The Probelm is that the height is not correct, the spacing at top and bottom are not the same.

<img width="1464" alt="Bildschirmfoto 2024-08-01 um 09 00 15" src="https://github.com/user-attachments/assets/69acb5af-8f5c-451c-925c-42042f77824f">

Reply:
I just checked, --sk-dialog-width and height is used, you can see the difference between dev and this PR:
![image](https://github.com/user-attachments/assets/5d157956-18fd-482a-9bd5-d4f4c277c3ea)

Reply:
we fixed is, only thing that is different now is that the default dialog dont take the full width but fits the content. But we thought it is better to have fit content by default. 

(left = before, right = now)
<img width="1913" alt="Bildschirmfoto 2024-08-01 um 10 49 10" src="https://github.com/user-attachments/assets/a596ecee-e2bf-45b5-bec4-9995289ab693">